sdk/iOS/OoyalaSkinSDK/OOSkinViewController.h

```diff
@@ -22,6 +22,24 @@
 @property (nonatomic, readonly) NSString *version;
 @property (nonatomic, readonly) OOClosedCaptionsStyle *closedCaptionsDeviceStyle;
 
+/**
+ Programatically change the fullscreen mode of the player.
+ Use this only after setting the OOSkinViewController.view.frame. Using it before will result in unexpected results.
+ 
+ Here is an example on how to use this property:
+ @code
+ // assume the property skinController of type OOSkinViewController exists and is already instantiated.
+ // Also assume you have an IBOutlet to a self.videoView pointing to an existing view.
```

Comment:
OOSkinViewController should exist always to use the fullscreen property, since it is part of it.

About self.videoView, in the example is needed because we need to set the frame of the skinViewController.view to a know value by the user. In the example it is assuming that there exists a view to which we reference through self.videoView, let's say that it's of size 100x100, so the player view will be of this size and positioned where the self.videoView is. The user can decide to set the frame with the value they want, but they must know that it has to be visible in the screen.

If they don't set the value of the SkinViewController.view.frame before calling to fullscreen, they will see an unexpected UI for the player, since it will use the default value of the SkinViewController.view.frame when trying to go fullscreen, since fullscreen depends on SkinViewController.view.frame.